kpconv_torch/utils/data_functions.py

```diff
@@ -1,6 +1,7 @@
 import sys
```

Comment:
Renamed file: `ply.py` => `data_functions.py`
Functions are alphabetically reordered
New functions added: read_xyz, read_las_laz
Function modified: read_ply

Reply:
What about `io.py`? We are talking about reader/writer utilities, it should be understandable at the first glance.

Reply:
Or better: a subfolder `io` which contains separate module `ply.py`, `las.py`, `xyz.py`, ...

Reply:
That's not only a matter of size, but also a matter of meaning: the module hierarchy should highlight our *intention*.

Reply:
ok, chosen the second one finally.